### ISSUE TITLE
java.lang.NullPointerException: at com.instabug.bug.view.reporting.b.initViews

### ISSUE THREAD

**Issue by mrarronz** [Instabug/Instabug-React-Native#469]
Instabug crashes on Android 10, this crash both recorded in google play console and firebase console

**Expected Behavior**
Instabug should not crash

**Actual Behavior**
Instabug crashes

**SDK Version**
Instabug-reactnative:v8.7.3.1, this is a customBuild from @alyezz 

**React Native, iOS and Android Versions**
React Native: 0.61.2
buildToolsVersion = "28.0.3"
minSdkVersion = 21
compileSdkVersion = 28
targetSdkVersion = 28

**Device Model**
Google Pixel 3a XL, Android 10

Here is the crash log:
```
java.lang.NullPointerException: 
  at com.instabug.bug.view.reporting.b.initViews (BaseReportingFragment.java:262)
  at com.instabug.library.core.ui.InstabugBaseFragment.onCreateView (InstabugBaseFragment.java:38)
  at androidx.fragment.app.Fragment.performCreateView (Fragment.java:2600)
  at androidx.fragment.app.FragmentManagerImpl.moveToState (FragmentManagerImpl.java:881)
  at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState (FragmentManagerImpl.java:1238)
  at androidx.fragment.app.FragmentManagerImpl.moveToState (FragmentManagerImpl.java:1303)
  at androidx.fragment.app.FragmentManagerImpl.dispatchStateChange (FragmentManagerImpl.java:2659)
  at androidx.fragment.app.FragmentManagerImpl.dispatchActivityCreated (FragmentManagerImpl.java:2613)
  at androidx.fragment.app.FragmentController.dispatchActivityCreated (FragmentController.java:246)
  at androidx.fragment.app.FragmentActivity.onStart (FragmentActivity.java:542)
  at androidx.appcompat.app.AppCompatActivity.onStart (AppCompatActivity.java:200)
  at com.instabug.library.core.ui.BaseFragmentActivity.onStart (BaseFragmentActivity.java:51)
  at com.instabug.bug.view.reporting.ReportingContainerActivity.onStart (ReportingContainerActivity.java:118)
  at android.app.Instrumentation.callActivityOnStart (Instrumentation.java:1432)
  at android.app.Activity.performStart (Activity.java:7848)
  at android.app.ActivityThread.handleStartActivity (ActivityThread.java:3294)
  at android.app.servertransaction.TransactionExecutor.performLifecycleSequence (TransactionExecutor.java:221)
  at android.app.servertransaction.TransactionExecutor.cycleToPath (TransactionExecutor.java:201)
  at android.app.servertransaction.TransactionExecutor.executeLifecycleState (TransactionExecutor.java:173)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:97)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2016)
  at android.os.Handler.dispatchMessage (Handler.java:107)
  at android.os.Looper.loop (Looper.java:214)
  at android.app.ActivityThread.main (ActivityThread.java:7356)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:492)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:930)
```

**Comment by mrarronz** [comment]
I've searched on https://github.com/Instabug/Instabug-Android/issues/190, from the firebase we can see that this is the same crash. Here is the log from firebase:
```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'com.instabug.library.model.State com.instabug.bug.model.a.getState()' on a null object reference
       at com.instabug.bug.view.reporting.b.initViews(BaseReportingFragment.java:262)
       at com.instabug.library.core.ui.InstabugBaseFragment.onCreateView(InstabugBaseFragment.java:38)
       at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2600)
       at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:881)
       at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManagerImpl.java:1238)
       at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:1303)
       at androidx.fragment.app.FragmentManagerImpl.dispatchStateChange(FragmentManagerImpl.java:2659)
       at androidx.fragment.app.FragmentManagerImpl.dispatchActivityCreated(FragmentManagerImpl.java:2613)
       at androidx.fragment.app.FragmentController.dispatchActivityCreated(FragmentController.java:246)
       at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:542)
       at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:200)
       at com.instabug.library.core.ui.BaseFragmentActivity.onStart(BaseFragmentActivity.java:51)
       at com.instabug.bug.view.reporting.ReportingContainerActivity.onStart(ReportingContainerActivity.java:118)
       at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1432)
       at android.app.Activity.performStart(Activity.java:7848)
       at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3294)
       at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:221)
       at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:201)
       at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:173)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:214)
       at android.app.ActivityThread.main(ActivityThread.java:7356)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

**Comment by mrarronz** [comment]
Hello @alyezz This crash is happening on the version https://github.com/Instabug/Instabug-React-Native#customBuild/8.7.3.1 you have posted to me in issue #467 .  

Maybe that fixed the crash mentioned in issue #467 , but a new crash occurred, could you please help?? Since this crash happened just on 1 user on our local testing, if we push that to production, the crash rate must be increasing quickly as the same. 

Is it possible to make a fix based on this custom version again? We are worried about this, could you please help??? Thank you very much.

**Comment by mrarronz** [comment]
Hi @mzelzoghbi , I see that you have made a fix on Android part about this crash in previous release, could you please help to get this fixed on the customBuild/8.7.3.1?? Or could you tell me how to fix it? Really appreciate your help.

**Comment by alyezz** [comment]
@mrarronz i created another custom build for you on a new branch `customBuild/8.7.3.2`containing the fix for this crash.
It is recommended however to update to the latest version as it contains alot of fixes.

**Comment by mrarronz** [comment]
Thank you! @alyezz 
We have released our app with customBuild/8.7.3.1, looks good than I thought. This crash seems not happen for real users, only occurred once during our testing. But Instabug library still got some OutOfMomory crashes,  the OOM crashes are always there since we integrated  instabug-reactnative, it's been a legacy problem.

Anyway, thank you for your help!
